### DISCUSSION TITLE
ruby 3.3.6 availability

### DISCUSSION THREAD

**Discussion by balakirevs** [rbenv/ruby-build#2459] [General]
Hi, could u pls add ruby 3.3.6?

**Comment by mislav** [comment]
Hi, it will be possible to add Ruby 3.3.6 only after it becomes a publicly available release.

**Comment by balakirevs** [comment]
sorry my bad, seeing release  Ruby 3.2.6 | 2024-10-30 and thinking about 3.3.6 🤣

**Reply by mislav** [reply]
Thanks for clarifying. Ruby 3.2.6 got added to ruby-build on the same day: https://github.com/rbenv/ruby-build/releases/tag/v20241030
